Makefile

```
@@ -0,0 +1,27 @@
+ENTRYPOINT=cmd/app/main.go
```

все таргеты, которые не создают файл по имени таргета должны быть добавлены в зависимости цели .PHONY
```suggestion
.PHONY: build run test test_with_coverage cleantest retest doc
ENTRYPOINT=cmd/app/main.go
```